### ISSUE TITLE
Multi Action Button children not contained by Landmarks. 

### ISSUE THREAD

**Issue by JohnOxleySage** [Sage/carbon#6664]
### Description

I am using the Multi Action button inside of a header, however I am getting an Axe error when the dropdown is open. The Axe error is to do with not all content being contained by landmarks. This is because the contents of the dropdown are created outside of the container the button is being rendered in. 

### Reproduction

https://stackblitz.com/edit/parsium-carbon-starter-1tcaax

### Steps to reproduce

Open the preview in a new tab, open the dropdown and run an Axe report. 

### Suggested solution

The content of the dropdown should be created inside the landmark. 

### Carbon version

v126.4.0

### Design tokens version

4.34.0

### Relevant browsers

Chrome

### Relevant OSs

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
Hi @JohnOxleySage, this is a moderate issue where the content of the dropdown is being rendered in a Portal elsewhere in the DOM. I will confirm with accessibility if this is an issue that needs addressing immediately or if this is more of a "it would be nice if it worked a different way but it won't stop anyone using it" kind of thing.

**Comment by nicktitchmarsh** [comment]
I will close this issue for now as there isn't a priority to fix this at the moment.
